test/tasks/demo.test.js

```diff
@@ -126,36 +129,34 @@ describe('Demo task', function() {
 			});
 		});
 
-		it('should build demo html', function(done) {
+		it('should build demo html', function() {
 			fs.writeFileSync('demos/src/test1.mustache', '<div>test1</div>', 'utf8');
 			fs.writeFileSync('demos/src/test2.mustache', '<div>test2</div>', 'utf8');
-			demo(gulp)
+			return demo(gulp)
 				.then(function() {
 					var test1 = fs.readFileSync('demos/test1.html', 'utf8');
-					var test2 = fs.readFileSync('demos/test1.html', 'utf8');
+					var test2 = fs.readFileSync('demos/test2.html', 'utf8');
```

Comment:
I actually broke this in this PR:- https://github.com/Financial-Times/origami-build-tools/pull/193/files#diff-6a5945ca98ba77b07f1c75a9d4888aebR135

Which shows even though Travis has been telling us things are good, the tests haven't actually been passing for a few days…

I don't think the `npmTest` task works properly :-) @AlbertoElias